sdk/iOS/test/MSSyncTableTests.m

```diff
@@ -1584,6 +1584,142 @@ -(void) testPullWithFetchLimitNotMultipleOfFifty
                   @"Invalid URL: %@", thirdRequest.URL.absoluteString);
 }
 
+-(void) testPullTable_UsePullSettings_NegativePageSize
+{
+    MSPullSettings *pullSettings = [MSPullSettings new];
+    pullSettings.pageSize = -1;
+    
+    NSString *pullRequest = [self getPullTableRequestWithPullSettings:pullSettings usePullSettings:YES];
+
+    XCTAssertEqualObjects(pullRequest, @"https://someUrl/tables/TodoNoVersion?$top=50&__includeDeleted=true&$skip=0&__systemProperties=__deleted", "Incorrect pull request");
+}
+
+-(void) testPullTable_UsePullSettings_ZeroPageSize
```

Comment:
I think these tests could just be moved to a test class for MSPullSettings and verify just it.  No need to go through the whole pipeline on them.